asb-ballerina/Ballerina.toml

```diff
@@ -2,7 +2,7 @@
 distribution = "2201.2.1"
```

Comment:
As this goes to 2.x branch, change ballerina version in all the places to 2201.2.3